packages/lodestar/test/unit/eth1/impl/ethers.test.ts

```diff
@@ -61,6 +61,7 @@ describe("Eth1Notifier", () => {
     provider.getBlockNumber.resolves(currentBlockNumber);
 
     await eth1.start();
+    await Promise.resolve();
```

Comment:
a stray promise.resolve?

Reply:
remove it will call the test fail as `eth1` now does not await for `startProcessEth1Blocks`. This is like a hacky way but now sure how can we make it better.